yang/ietf-ucl-acl.yang

```diff
@@ -44,14 +44,72 @@ module ietf-ucl-acl {
       "RFC XXXX: A Policy-based Network Access Control";
   }
 
-  feature match-on-endpoint-group {
+  identity group-acl-type {
+    if-feature "group";
+    base acl:acl-base;
+    description
+      "An ACL that matches based on an abstract group identity,
+       which may be within the NVO3 header extension, but may not
```

Comment:
Why nvo3 specifically?

Reply:
If the intent is to cover encap schemes, e.g., outer/inner headers, then I would make make that explicit. Thanks.

Reply:
I have updated with: An endpoint
       group identity may be carried in the outer/inner packet
       header(e.g., via NVO3 encapsulation), but may not correspond
       to any field in the packet header.
Any suggestion?

Reply:
Works for me.  thanks.